DEVELOPMENT.md

```diff
@@ -183,6 +186,7 @@ By default, KubeArchive listens to `Event`s in the `test` namespace.
     ```bash
     go run cmd/kubectl-archive/main.go get batch/v1 jobs --token $(kubectl create -n test token default)
     ```
+   **NOTE**: For this to work the `test/users/test-user.yaml` must be applied
```

Comment:
You need a period at the end of the sentence:

```suggestion
   **NOTE**: For this to work the `test/users/test-user.yaml` must be applied.
```

Reply:
Fixed.